test/reporters/verbose.timeoutinmultiplefiles.log

```diff
@@ -5,9 +5,9 @@
   [32m✔[39m a [90m[2m›[22m[39m a passes two
 ---tty-stream-chunk-separator
   [31m[39m
-  [31m✖ Exited because no new tests completed within the last 10000ms of inactivity[39m
+  [31m✖ Timed out while running tests[39m
```

Comment:
The original copy seems better for the `--timeout` scenario. What made you change it?

Reply:
It seemed a bit long, repetitive and unclear.
Exited - not always true
no new tests - why the `new`?
10000ms - do I have to pull out a calculator if its longer duration? do I actually not know AND care about what the duration was?
of inactivity - repetitive